### PR TITLE
WIP : content and design features

### PR DESCRIPTION
**Pull request by fuentesloic**
following the issue #129

### CODE REVIEW

**Comment by fuentesloic** [comment]
capture d ecran 2017-07-14 a 18 02 15-fullpage

**Comment by AymericFaivre** [comment]
Nice job 👍 
It's perfect to me :)

File: templates/home.html [hunk 1]
```diff
@@ -29,10 +29,49 @@
     <h2>Find out contents for nomad</h2>
     <h3>Inspire and be inspire!</h3>
     <ul>
-      <li><p>Read, write and contribute with nomad community</p></li>
-      <li><p>Follow your nomad life on your own blog and map</p></li>
-      <li><p>Find other nomad close to you and enjoy with them</p></li>
-      <li><p>Find and share a place to work anywhere &amp; anytime</p></li>
+      <li>
+        <div class="content">
```
Comment (Ydrasil) [inline templates/home.html]: Maybe you can remove the double quote here

File: templates/home.html [hunk 2]
```diff
@@ -29,10 +29,49 @@
...
+          </ul>
+          <div>
+            <a href=# class=button>Read from nomads</a>
+            <a href=# class=button>Write your stories</a>
+          </div>
+        </div>
+        <img src={{ url_for('static', filename='images/features/articles.png') }} alt='read articles'>
+      </li>
+      <li>
+        <div class="content">
```
Comment (Ydrasil) [inline templates/home.html]: Same here about the double quote

File: templates/home.html [hunk 3]
```diff
@@ -29,10 +29,49 @@
...
+            <li>Follow travelers on their map and get inspired!</li>
+          </ul>
+          <div>
+            <a href=# class=button>Start sharing your trips</a>
+          </div>
+        </div>
+        <img src={{ url_for('static', filename='images/features/trips.png') }} alt='read articles'>
+      </li>
+      <li>
+        <div class="content">
```
Comment (Ydrasil) [inline templates/home.html]: Here too

File: templates/home.html (hunk 1 above)
Comment (fuentesloic) [reply templates/home.html]: true ! I forget all the time :(

**Comment by newick** [comment]
It looks great for me!

**Comment by vinyll** [comment]
@all: I think @fuentesloic is taking a good direction and we could help him getting something challenging and making a good UX.

Let's not forget this is the landing page, meaning the one page everyone will see, and the only page that most people see as they will leave the site after the 1st page (most sites have users doing that). Meaning this is the most important page.


## Section 2

- Is having 3 times a book icon for the 3 sections pertinent? If so what are they there for?
- Is the green color appropriate?
- Are the call-to-actions buttons appropriate in their design/position/message?
- Are the images that are aside the text explaining the text and attractive?
- Is "find out contents for nomad" a user-friendly title?


## Section 3

- Are articles displayed like this fine for a homepage?
- What have other do that is great?

The website will soon be open to the full public. Over 100 persons have asked to know more and are about to be contacted back.
If you actually feel this is perfect, let's merge this. If you think there's any room for a step further for the landing page, please share it out loud 📢

**Comment by AymericFaivre** [comment]
Yeah, in fact, there is few things (I looked very quickly)

General : Maybe we can try with the font I spotted last time

Part 1 : The font is always bold. Can we make variations to have more contrast?

Part 2 : 
-Each block content need alignement.
- We could use diferents icons for each part
- I'm not sure about the screenshot. It's maybe too raw, or maybe it's not necessary.
- Maybe we could try the new fonts?
- Maybe Track  and around you could be on the same part (to me it's the same. You have the possibility to tack your postion, and you can share it).
- Buttons coud be more obvious (green border, green font)
-> This part need more work but I have no magic idea for the moment (hope I will have some time in august)

Part 3 : 
- In fact, @vinyll is right, I like the presentation in these 2 websites 
http://cheerup.theme-sphere.com/magazine/#post-853
http://sites.layerswp.com/rayne/
Its's cleaner.


Sorry to don't have time yet to make a mockup of all of that :/
Good luck with that !

**Comment by fuentesloic** [comment]
Just a step inspired by your reviews (not sure if it's usefull ...)
capture d ecran 2017-07-18 a 17 52 04-fullpage

**Comment by newick** [comment]
Time for me to be relou :)

I've got issues with the buttons.
Once they have border-radius, in the other section they dont. In article they are full width, the other time not.
It would be better if we harmonize everything, isn't it?

A great solution could be to create a basic styleguide / pattern librairy with basic styles like headers, buttons, etc.

**Comment by fuentesloic** [comment]
@newick oh yes !! a styleguide could be THE solution for me :)

**Comment by fuentesloic** [comment]
Simple, clear and (I hope) clean
Next step the font
Then style guide (other branch & PR)
capture d ecran 2017-07-20 a 09 28 50-fullpage

**Comment by newick** [comment]
i'm really not fan of the 100% width buttons in articles list. 
Maybe a min-width?!

Any tips @AymericFaivre?

**Comment by fuentesloic** [comment]
with new font and cleaned stuff :p
capture d ecran 2017-07-20 a 09 57 24-fullpage

**Pull request merged**